src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Home/Index.cshtml

```diff
@@ -6,3 +6,7 @@
     <h1 class="display-4">Welcome</h1>
     <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
 </div>
+
```

Comment:
WE probably want to guard it with calls graph or API?

Reply:
Sure I can add it. Wasn't sure if that was needed since it's not in the templates I copied from https://github.com/AzureAD/microsoft-identity-web/blob/master/ProjectTemplates/templates/StarterWeb-CSharp/Views/Home/Index.cshtml